src/providers-registry/types.ts

```diff
@@ -139,6 +139,10 @@ export interface AddProviderManifestData {
   lockedHours?: number;
 }
 
+export interface GetProviderAdmins {
```

Comment:
should be also added to data property of ProvidersRegistryInput in `src/providers-registry/types.ts`